profile_library/signify/LDT001/model.json

```
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-04-18T08:54:05.495469",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly, 1PM Mini Gen 3",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.17.10:docker"
+  },
+  "name": "Hue ambiance downlight (LDT001)",
```

You may use the marketed name here.
Will be shown on library website: https://library.powercalc.nl in name column.

```suggestion
  "name": "Hue Aphelion Recessed Downlight White Ambiance",
```